src/server/render-widget.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
```

Comment:
Think it's diverging too much if we used `preact` for the widget? @artibella @eschaefer?